nifi-commons/nifi-record-path/src/test/java/org/apache/nifi/record/path/TestRecordPath.java

```diff
@@ -1644,6 +1646,48 @@ public void testPadRight() {
         assertEquals("MyStringfewfewfewfew", RecordPath.compile("padRight(/someString, 20, \"few\")").evaluate(record).getSelectedFields().findFirst().get().getValue());
     }
 
+    @Test
+    public void testUuidV5() {
+        final List<RecordField> fields = new ArrayList<>();
+        fields.add(new RecordField("input", RecordFieldType.STRING.getDataType()));
+        fields.add(new RecordField("name", RecordFieldType.STRING.getDataType(), true));
+        final RecordSchema schema = new SimpleRecordSchema(fields);
+        final UUID name = UUID.fromString("67eb2232-f06e-406a-b934-e17f5fa31ae4");
+        final String input = "testing NiFi functionality";
+        final Map<String, Object> values = new HashMap<>();
+        values.put("input", input);
+        values.put("name", name.toString());
```

Comment:
Not a big deal but this is a little confusing since name usually refers to the input.

Reply:
I changed `name` to `namespace`.